Kitodo/src/main/java/de/sub/goobi/forms/AktuelleSchritteForm.java

```diff
@@ -135,7 +135,7 @@ public String filterAll() {
             Helper.setFehlerMeldung("Error on reading ElasticSearch: ", e.getMessage());
             return null;
         }
-        return "/pages/AktuelleSchritteAlle";
+        return redirectToList("?faces-redirect=true");
```

Comment:
Please change your code like the changes from @solth  in `BenutzergruppenForm`, `DocketForm` or `ProjekteForm`: use the constant value from `BasisForm` and add this value inside the method and not as a method parameter.